tests/unit/deleteTestResultsList.unitTest.ts

```diff
@@ -1,88 +0,0 @@
-import { TestResultsService } from "../../src/services/TestResultsService";
```

Comment:
Why did you have to remove the 2 files with unit tests?

Reply:
The deleteTestResultsList and the insertTestResultList is not used anywhere. I ran the backend regression suite after removing these functions and nothing failed. Hence the tests related to the code also need to be removed.